checkstyle-tester/diff.groovy

```diff
@@ -818,7 +805,7 @@ class Config {
             checkstyleCfg: baseConfig,
```

Comment:
@romani What are these `getCheckstyleToolBaseConfig` for? No one calls them in this script.

Reply:
might be dead code - please remove.

Reply:
Not dead code.
https://github.com/checkstyle/contribution/pull/758#issuecomment-1369098505